src/redux/categories/categories.jsx

```diff
@@ -1,7 +1,17 @@
 import { createSlice } from '@reduxjs/toolkit';
 
 const initialState = {
-  categories: [],
+  categories: [
+    'Fantasy',
+    'Sci-Fi',
+    'Mystery',
+    'Thriller',
+    'Romance',
+    'Westerns',
+    'Dystopian',
+    'Contemporary',
+    'Action',
+  ],
 };
 
 const categoriesSlice = createSlice({
```

Comment:
- When I go to the categories page and click the `Check Status` button, it breaks the home page.
It is happening because the check status reducer is manipulating the state that is being used on the home page for categories.

- You can fix this bug by keeping the original state using the spread method.
```
checkStatus: () => ({ ...initialState, state: 'Under Contrustion' }),
```
Good Luck :)